### ISSUE TITLE
[1.3.5.3, Linux] Weird Input issues

### ISSUE THREAD

**Issue by asdfgh110** [bluemagic123/tModLoader#2015]
### Version

Latest Official

### OS

Linux

### Platform

Steam

### Description

When I hold Space, I keep jumping. Tried with and without mods, the issue persists. Can't seem to reproduce on 1.4 Alpha. Also works with going left and right (A, D). It seems that when I hold a button, it's getting spammed very inconsistently?

### Steps to reproduce

1. Hold Space
2. Jump repeadetly

### Expected Behavior

Jump only once

### Actual Behavior

Continuous jumps

### Reproduction frequency

100% of the time

### Additional Information

https://user-images.githubusercontent.com/88080558/144441322-113ac7a0-5b2e-4fe1-bb44-78853b3db14b.mp4

**Comment by uuigdevt47** [comment]
Try `xset r off` if you are using X11.

**Comment by RunningDroid** [comment]
Some Archlinux users appear to have come across a similar [issue](https://wiki.archlinux.org/title/Steam/Game-specific_troubleshooting#Input_Issues); Try adding `LD_PRELOAD='/usr/$LIB/libSDL2-2.0.so:/usr/lib32/libSDL2-2.0.so' %command%` to tModLoader's launch options like this:

![2021-12-22 02:18:29](https://user-images.githubusercontent.com/486393/147053336-0865ffe6-4d3b-4dbd-b042-6448870abbf5.png)

**Comment by jmalionek** [comment]
Also having this issue (but am on Ubuntu) and neither of the above solutions worked
UPDATE: xset r off works but only if I do the command AFTER I've started up tmodloader 1.3. (It appears that TmodLoader resets it to be on when it starts up for me?)

**Comment by JavidPack** [comment]
This issue relates to 1.3 tModLoader, which is now legacy and will not receive any further non-critical updates or bug fixes. Please give 1.4 tModLoader a try, there is a good chance this issue might be resolved there already. If you find that 1.4 tModLoader has a similar issue, feel free to open a new issue and fill out the issue template so we can once again track this issue. This issue will now be closed.
